### ISSUE TITLE
the timing is too optimistic

### ISSUE THREAD

**Issue by wikfeldt** [coderefinery/git-solo#13]
episodes from "basics" up until "sharing repositories online" can take from 9:30 to 14:30

**Comment by Sabryr** [comment]
This is true. It also dependent on audience. Anyway rushing this initial part is not good and the mentality build will affect rest of the course.
